php/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM microsoft/iis:windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV PHP_VERSION 7.1.0
+ENV PHP_SHA1 85c0215337b08f9d49c233c16475c15f17b66ac1
+
+WORKDIR /Users/ContainerAdministrator/Downloads
+
+# Visual C++ 2015 Redistributable
+# workaround: the .exe command completes early and does not finish the install, so added 10s sleep to the end to allow time to finish
+RUN Invoke-WebRequest 'https://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe' -OutFile 'vc_redist.x64.exe'; \
+	.\vc_redist.x64.exe /install /passive /norestart; \
+    Start-Sleep -s 10;
+
+# Install PHP
+RUN Invoke-WebRequest $('http://windows.php.net/downloads/releases/php-{0}-nts-Win32-VC14-x64.zip' -f $env:PHP_VERSION) -OutFile 'php.zip'; \
+    if ((Get-FileHash php.zip -Algorithm sha1).Hash -ne $env:PHP_SHA1) {exit 1} ; \
+    Expand-Archive -Path php.zip -DestinationPath c:\php; \
+    [Environment]::SetEnvironmentVariable('PATH', $env:Path + ';C:\php', [EnvironmentVariableTarget]::Machine); \
+	$env:PATH = [Environment]::GetEnvironmentVariable('PATH', [EnvironmentVariableTarget]::Machine); \
+    php --version;
+
+COPY stage/ /
+
+# Install Composer
+RUN Invoke-WebRequest 'https://getcomposer.org/installer' -OutFile 'composer-setup.php'; \
+    php composer-setup.php; \
+    mv composer.phar C:\php; \
+    composer about;
+
+# Configure IIS
+COPY config/ ./
+RUN .\configure_iis.ps1;
+
+# Clean Downloads Directory
+RUN Remove-Item -Recurse -Force *

```

Comment:
Removing files in a separate `RUN` command does not reduce the size of the layers to pull from a container registry as all layers have to be pulled to build the Docker image. If a first `RUN` downloads a GByte and a second `RUN` deletes it, the first layer is still a GByte.

So it would be better to remove the files at the end of the corresponding `RUN` command.